src/Extension/ValueResolver/PriorityPathValueResolver.php

```diff
@@ -53,12 +61,13 @@ public function getValue(ContextInterface $context, array $settingValues, bool $
                 $path = rtrim($path, DIRECTORY_SEPARATOR);
             }
             $path = implode(DIRECTORY_SEPARATOR, [$path, $relativePath]);
+
             if (file_exists($path)) {
                 return $this->formatValue($path);
             }
         }
 
-        throw new UnresolvableValueExceptionException('Can\'t resolve path.');
+        throw new UnresolvableValueExceptionException('Invalid path provided.');
```

Comment:
Normally it is better to have success scenario at the end. For this reason and for the readability perspectives, I'd recommend extracting `foreach` loop above to the separate method `extractFormattedPathValue()` and do the following here instead
```php
$pathValue = $this->extractFormattedPathValue($settingValues);
if (!$pathValue) {
     throw new UnresolvableValueExceptionException('Invalid path provided.');
}

return $pathValue;